### PR TITLE
Updates bridge to run on Java 11.

### PR DESCRIPTION
**Pull request by dbernstein**
Resolves: https://jira.lyrasis.org/browse/DURACLOUD-1274

### CODE REVIEW

File: snapshot-service-impl/src/main/java/org/duracloud/snapshot/service/impl/RestoreManagerImpl.java [hunk 1]
```diff
@@ -114,6 +114,14 @@ public void setStoreClientHelper(StoreClientHelper storeClientHelper) {
         this.storeClientHelper = storeClientHelper;
     }
 
+    /**
+     *
+     * @param eventLog the event log to set
+     */
+    public void setEventLog(EventLog eventLog) {
```
Comment (bbranan) [inline snapshot-service-impl/src/main/java/org/duracloud/snapshot/service/impl/RestoreManagerImpl.java:121]: Is this added only to support a test? Is there really a need to capture events in the test context. In any case if these are only for tests they should be protected scope and the javadoc should indicate this as well (to make clear why the method isn't in the RestoreManager interface).

File: snapshot-service-impl/src/main/java/org/duracloud/snapshot/service/impl/SnapshotManagerImpl.java [hunk 2]
```diff
@@ -512,4 +512,11 @@ private Snapshot changeSnapshotStatus(Snapshot snapshot,
         return savedSnapshot;
     }
 
+    /**
+     *
+     * @param eventLog the event log
+     */
+    public void setEventLog(final EventLog eventLog) {
```
Comment (bbranan) [inline snapshot-service-impl/src/main/java/org/duracloud/snapshot/service/impl/SnapshotManagerImpl.java:519]: As above, is this needed only for tests?

File: snapshot-service-impl/src/main/java/org/duracloud/snapshot/service/impl/RestoreManagerImpl.java [hunk 3]
```diff
@@ -522,4 +530,7 @@ public Restoration restartRestore(String restoreId) throws SnapshotException {
         return this.restoreCompleted(restoration);
     }
 
+    public void setSnapshotManager(final  SnapshotManager snapshotManager) {
```
Comment (bbranan) [inline snapshot-service-impl/src/main/java/org/duracloud/snapshot/service/impl/RestoreManagerImpl.java:533]: As above, is this needed only for tests?

File: snapshot-service-impl/src/main/java/org/duracloud/snapshot/service/impl/StepExecutionSupport.java [hunk 4]
```diff
@@ -128,6 +128,7 @@ protected void failExecution() {
      */
     @Override
     public ExitStatus afterStep(StepExecution stepExecution) {
+        this.stepExecution = stepExecution;
```
Comment (bbranan) [inline snapshot-service-impl/src/main/java/org/duracloud/snapshot/service/impl/StepExecutionSupport.java:131]: Was this a bug? It doesn't seem related to the Java 11 transition.

**Review by bbranan — COMMENTED** [review]
I built this and ran it with Java11 and it worked well, so no issues there. I had a few questions on the code, mostly about added methods.

File: snapshot-service-impl/src/main/java/org/duracloud/snapshot/service/impl/StepExecutionSupport.java (hunk 4 above)
Comment (dbernstein) [reply snapshot-service-impl/src/main/java/org/duracloud/snapshot/service/impl/StepExecutionSupport.java:131]: It was causing a problem with one of the tests.  Probably related to the easymock upgrade.
